### ISSUE TITLE
Add Streamlit streamming of output 

### ISSUE THREAD

**Issue by neokd** [neokd/NeoGPT#53]
The bot doesn't show proper response in the UI and there is also no streamming support. Add one in UI.py

**Comment by HadassaAvimor** [comment]
Hey @neokd ,
I would like to do it, please assign this issue for me.
Thanks!

**Comment by neokd** [comment]
@HadassaAvimor i have assigned the issue to you. Any updates on it?

**Comment by HadassaAvimor** [comment]
@neokd I have some sudden engagements in the last few days, which will make me more busy in the coming days. I'll try to do it as soon as possible, but you can assign this issue to something else for now, and I'll take on another issue from the project when I have more time.

**Comment by neokd** [comment]
@HadassaAvimor yeah sure. If you have time you finish it who ever creates the PR I'll merge the code

**Comment by savyez** [comment]
hey, can you describe the issue a bit more? It seems interesting to me as my initial projects included streamlit library and I somewhat have knowledge of it.

**Comment by neokd** [comment]
@savyez there is a file called Ui.py which is used to make a basic UI for NeoGPT.
The file is used to create a basic chat interface for users to interact with the bot rather than always relying on CLI.The file is build using the old code and is not working anymore. Also in the old file there was no streaming support and the message would take 2-3 min to show up in the UI. Streamming helps used to view the model response instantly (something similar you can see with chatgpt or in the CLI of NeoGPT)

**Comment by savyez** [comment]
@neokd yeah thanks for the clarification, I would like to take some time for researching on what could be the most efficient way to solve this problem. May I have some time? (I would inform you within 1PM IST tomorrow)

**Comment by neokd** [comment]
Yeah sure @savyez

**Comment by savyez** [comment]
hey @neokd  I looked in this problem and Ui file seems to give some buggy error when executed.
`from neogpt.vectorstore.base import VectorStore
ModuleNotFoundError: No module named 'neogpt'`

idk if the protoype ui was fully built before the PR was raised.

**Comment by neokd** [comment]
@savyez you can remove `neogpt.` And just use `vectorstore.ChromaStore `

**Comment by savyez** [comment]
thank you  for the clarification, may I take it? if it is available?

**Comment by neokd** [comment]
Yeah sure
